src/MarketingPageAcceptanceTestsSpecflow/Steps/ClientApplication/NativeMobileOrTablet/OperatingSystemsSupported.cs

```diff
@@ -10,7 +10,7 @@ public OperatingSystemsSupported(UITest test, ScenarioContext context) : base(te
         {
         }
 
-        [Given(@"that a User has provided a value for the Mandatory Information for (.*) section on (.*) sub dashboard")]
+        [Given(@"that a User has provided a value for the Mandatory Information for (Supported operating systems) section on (Native mobile or tablet) sub dashboard")]
```

Comment:
This would stop other sections and subsections from being able to use this method, which would make the switch statement and parameterisation useless